xml/yast2_ncurses.xml

```diff
@@ -352,48 +333,47 @@
    <para>
     <replaceable>package_name</replaceable> can be a single short package name
     (for example <package>gvim</package>) installed with dependency checking,
-    or the full path to an RPM package which is installed without dependency
+    or the full path to an RPM package, which is installed without dependency
     checking.
    </para>
    <para>
-    If you need a command line based software management utility with
-    functionality beyond what &yast; provides, consider using Zypper. This
-    utility uses the same software management library that is also the
-    foundation for the &yast; package manager. The basic usage of Zypper is
-    covered in <xref linkend="sec-zypper"/>.
+    While &yast; offers basic functionality for managing software from the
+    command line, consider using Zypper for more advanced package management
+    tasks. You will find more information on using Zypper in <xref
```

Comment:
To reduce complexity for non-native speakers of English, use present tense (or even imperative in this case) instead of future tense.
```suggestion
    tasks. Find more information on using Zypper in <xref
```